apps/backend/src/datasources/postgres/FapDataSource.ts

```diff
@@ -303,16 +303,14 @@ export default class PostgresFapDataSource implements FapDataSource {
           })
           .join('proposal_statuses as ps', {
             'p.status_id': 'ps.proposal_status_id',
-          })
```

Comment:
While I was testing I wanted to see all the proposals assigned to a FAP with any status. Maybe I can revert this change because for now we don't have any intentions for it.